src/Features/CSharp/Portable/CodeFixes/Suppression/CSharpSuppressionCodeFixProvider.cs

```diff
@@ -110,8 +111,7 @@ protected override SyntaxNode AddGlobalSuppressMessageAttribute(SyntaxNode newRo
             bool needsLeadingEndOfLine)
         {
             var attributeArguments = CreateAttributeArguments(targetSymbol, diagnostic);
-            var attribute = SyntaxFactory.Attribute(SyntaxFactory.ParseName(SuppressMessageAttributeName), attributeArguments)
-                .WithAdditionalAnnotations(Simplifier.Annotation);
+            var attribute = SyntaxFactory.Attribute(SyntaxFactory.ParseName(SuppressMessageAttributeName), attributeArguments);
```

Comment:
What API can I use to get the minimal type name? `SuppressMessageAttributeName` = `System.Diagnostics.CodeAnalysis.SuppressMessage`.